src/input/read.c

```diff
@@ -56,12 +56,15 @@ static str_t *handle_not_tty(void)
 void read_input(shell_t *state)
 {
     str_t *temp;
+    int ignore;
 
     while (!state->stop_shell) {
         if (!state->is_atty) {
             temp = handle_not_tty();
         } else {
-            temp = stock_input();
+            ignore =
+                atoi(((str_t *)map_get(state->vars, STR("ignoreof")))->data);
```

Comment:
to peux utiliser str_toint pour eviter devoir cast pour acceder a la data

Reply:
ah mdr